client/me/purchases/manage-purchase/purchase-meta.jsx

```diff
@@ -228,7 +228,6 @@ function PurchaseMetaPrice( { purchase } ) {
 	const translate = useTranslate();
 	const { productSlug, productDisplayPrice } = purchase;
 	const plan = getPlan( productSlug ) || getProductFromSlug( productSlug );
-	let period = translate( 'year' );
```

Comment:
Before this change, if no period could be found, it defaulted to `'year'`. After this change you'll get a fatal error if it could not determine the term. Was it intentional to change that behavior?

Reply:
Yes, you gave that as a suggestion [in this PR](https://github.com/Automattic/wp-calypso/pull/67021#pullrequestreview-1089313385)

`Don't assume that the default term is "yearly"; if we cannot figure out the term in any other way, maybe display nothing at all and/or throw an error.`

Based on the above suggestions, I decided to display an error. Let me know if that behaviour needs to be changed. @sirbrillig

Reply:
Ah, right. I forgot about that. And it's even written in the PR description!